fvm/src/machine/engine.rs

```diff
@@ -10,13 +10,19 @@ use fvm_ipld_blockstore::Blockstore;
 use fvm_wasm_instrument::gas_metering::GAS_COUNTER_NAME;
 use fvm_wasm_instrument::parity_wasm::elements;
 use wasmtime::OptLevel::Speed;
-use wasmtime::{Global, GlobalType, Linker, Memory, MemoryType, Module, Mutability, Val, ValType};
+use wasmtime::{
+    Global, GlobalType, InstanceAllocationStrategy, InstanceLimits, Linker, Memory, MemoryType,
+    Module, Mutability, PoolingAllocationStrategy, Val, ValType,
+};
 
 use crate::gas::WasmGasPrices;
 use crate::machine::NetworkConfig;
 use crate::syscalls::{bind_syscalls, InvocationData};
 use crate::Kernel;
 
+// Copied from `wasmtime_environ`.
+const WASM_PAGE_SIZE: u64 = 0x10000;
```

Comment:
nit: comment that this is 16k.

Reply:
Thanks, good point. It's actually 64KiB, so it's not intuitive at all. If you're okay with that, I'll add a dependency to `wasmtime_environ` itself to avoid duplicating it. The docs said that any crate other than `wasmtime` is considered "internal", and the comments on this suggested this is a WebAssembly specific value that won't change, but maybe this is the cleanest way to make sure we agree on the same value.